l10n_nl_tax_statement_icp/views/report_tax_statement.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018-2020 Onestein (<https://www.onestein.eu>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="action_report_tax_statement_icp" model="ir.actions.report">
+        <field name="name">NL ICP Statement</field>
+        <field name="model">l10n.nl.vat.statement</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">
+            l10n_nl_tax_statement_icp.report_tax_statement_icp
```

Comment:
The spaces here results in a traceback:
ValueError: External ID not found in the system:
l10n_nl_tax_statement_icp.report_tax_statement_icp

Reply:
@CasVissers-360ERP I can confirm teh exception. @astirpe Do you intend to take this up?

Reply:
After solving the report error, I do not see the customer lines in the generated report. Researching further...

Reply:
Aha, I first need to post the statement...

Reply:
@astirpe will you take up the comment from @CasVissers-360ERP - confirmed by me - on the spaces in the xml causing an error?